roles/cloud_connector/defaults/main.yml

```
@@ -0,0 +1,2 @@
+---
+cloud_connector_config_file: /etc/rhc/workers/rhc-worker-cloud-connector.toml
```

According to [#6](https://github.com/ShimShtein/yggdrasil-worker-forwarder/pull/6/files#diff-2873f79a86c0d8b3335cd7731b0ecf7dd4301eb19a82ef7a1cba7589b5252261R36), the directive name is derived from the config file name, hence:
```suggestion
cloud_connector_config_file: /etc/rhc/workers/foreman_rh_cloud.toml
```

---

Are you wanting this tied to foreman_rh_cloud rather than the cloud connector moniker?